generators/app/index.js

```diff
@@ -49,6 +49,7 @@ module.exports = Generator.extend({
 
   default: function () {
     var cmd = [
+      'rm -f master.tar.gz pattern-lab-starter-master patternlab',
```

Comment:
This behavior would seem rather unexpected though as I think through the scenarios it seems it is probably pretty safe. The only "data losing" scenario would be when someone has made changes inside of a patternlab directory and not subsequently committed them to their repo before re-running the generator and that seems fairly unlikely. It would be nice if we can detect that this is going to occur and prompt for confirmation but that doesn't seem like a blocker.